azul-css/src/css_properties.rs

```diff
@@ -631,6 +637,15 @@ pub enum StyleBackgroundSize {
     Cover
 }
 
+/// Represents a `background-repeat` attribute
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub enum StyleBackgroundRepeat {
```

Comment:
Make a `impl Default for StyleBackgroundRepeat` and then in the later code with the `Option<StyleBackgroundRepeat>`, just use `.unwrap_or_default()` (the default being `StyleBackgroundRepeat::Repeat`, because that's the default on the web). 
This way you can avoid the whole `if let Some(repeat) { }` and make the code a bit simpler.